java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/IceMX/MetricsHelper.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) ZeroC, Inc.
 
-package com.zeroc.Ice.IceMX;
+package com.zeroc.IceMX;
```

Comment:
Should we move this file to java/src/com/zeroc/ice/src/main/com/zeroc/IceMX/MetricsHelper.java ?